api/src/main/java/com/epam/pipeline/entity/metadata/CategoricalAttribute.java

```diff
@@ -16,13 +16,49 @@
 
 package com.epam.pipeline.entity.metadata;
 
+import com.epam.pipeline.entity.AbstractSecuredEntity;
+import com.epam.pipeline.entity.security.acl.AclClass;
+import lombok.Setter;
 import lombok.Value;
 
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Date;
 import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
 
 @Value
-public class CategoricalAttribute {
+@Setter
+public class CategoricalAttribute extends AbstractSecuredEntity {
 
-    private String key;
```

Comment:
Let's keep `key` field to simplify changes but set its value to `name` field as well since it is inherited from `BaseEntity `

Reply:
Done